### PR TITLE
Remove the .0 from the ends of vars

### PR DESCRIPTION
**Pull request by Evankhell561**
## About The Pull Request
Remove the .0 from the ends of vars

## Why It's Good For The Game

because someone had to do this.

### CODE REVIEW

File: code/__HELPERS/icons.dm [hunk 1]
```diff
@@ -5,7 +5,7 @@ A BYOND library for manipulating icons and colors
 
 by Lummox JR
 
-version 1.0
```
Comment (MadmanMartian) [inline code/__HELPERS/icons.dm:8]: lel

File: code/game/objects/items/devices/t_scanner.dm [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #define OVERLAY_CACHE_LEN 50
 
 /*
-	T-Ray scanners v2.0, By Nanako
```
Comment (Kurgis) [inline code/game/objects/items/devices/t_scanner.dm:4]: Similar to the other one pointed out.

File: code/modules/synthesized_instruments/globals.dm [hunk 3]
```diff
@@ -125,16 +125,16 @@ GLOBAL_DATUM_INIT(musical_config, /datum/musical_config, new)
 			"Value that controls the modal density in the late reverberation decay",
 			{"
 Bit flags that modify the behavior of above properties
-�1 - 'EnvSize' affects reverberation decay time
```
Comment (Kurgis) [inline code/modules/synthesized_instruments/globals.dm:128]: Corrupted text.

File: code/modules/maps/swapmaps.dm [hunk 4]
```diff
@@ -593,7 +593,7 @@ proc/SwapMaps_CreateFromTemplate(template_id)
 	/*
```
Comment (Kurgis) [inline code/modules/maps/swapmaps.dm:593]: Best leave the instructions intact, for whatever this is.

**Comment by SpadesSlick** [comment]
Someone should check if game runs properly after this PR. Those kind of changes are most dangerous.

File: code/__HELPERS/icons.dm (hunk 1 above)
Comment (TorinoFermic) [reply code/__HELPERS/icons.dm:8]: kek

**Pull request closed without merging**